build.sbt

```diff
@@ -2,7 +2,21 @@ name := "compiler-benchmark"
 
 version := "1.0-SNAPSHOT"
 
-scalaVersion in ThisBuild := "2.11.8"
+def scala211 = "2.11.11"
+def dottyLatest = "0.2.0-RC1"
+scalaVersion in ThisBuild := scala211
+
+commands += Command.command("testAll") { s =>
+  "test:compile" ::
+    "compilation/test" ::
```

Comment:
`+compilation/test` would not run on Dotty because it uses crossScalaVersions from the root project. Alternatives are

- `project compilation; +test ; project compiler-benchmark`
- add the sbt-doge plugin and run `such compilation/test`
- add 0.2.0-RC1 to compiler-benchmark/crossScalaVersions, but that would be incorrect.

I opted for duplication since it's the dumbest solution.